x-pack/legacy/plugins/xpack_main/public/services/path.js

```diff
@@ -6,9 +6,10 @@
 
 import chrome from 'ui/chrome';
 
+const paths = ['/login', '/logout', '/logged_out', '/status', '/oidc_implicit'];
```

Comment:
__note:__ we should find a better way of solving the underlying problem, but not in this PR likely.